dwl.c

```diff
@@ -224,6 +235,7 @@ static struct wlr_xdg_shell *xdg_shell;
 static struct wl_list clients; /* tiling order */
 static struct wl_list fstack;  /* focus order */
 static struct wl_list stack;   /* stacking z-order */
+static struct wl_list independents;
```

Comment:
This is a list of Client, containing all unmanaged surfaces e.g. dropdown menus.

We need to track this list so that we may render them.

dwm did not track these as X would handle them.